portfolio/src/main/webapp/about.html

```diff
@@ -157,6 +157,14 @@ <h3 class="section-subheading text-muted">We'd love to hear from you!</h3>
         </form>
       </div>
     </section>
+    <div id="googleForm">
+      <iframe
+        src="https://docs.google.com/forms/d/e/1FAIpQLSfN4mh5SLDzB2RXp2zMNTi_GSD-ywjBPOG4Scws5TKR7FlWcA/viewform?embedded=true"
+        title="Google Feedback Form"
+      >
+        Loading…
+      </iframe>
+    </div>
```

Comment:
I think this shouldn't be here.

Reply:
The unedited embedded link from the google form:

Reply:
I thought the Google Form was added in a separate PR.

Reply:
It was, but the base branch was to this branch

Reply:
You can rebase if need be. 
https://gist.github.com/loilo/930f141d9acf89e9e734ffa042acd750
After following those instructions you can force push the PR with the relevant commits.